### PR TITLE
Cu 1wuc2qe scrolling should hide artist bio

### PR DESCRIPTION
**Pull request by HenryQuillin**

### CODE REVIEW

**Comment by j-mateo** [comment]
Task linked: [CU-1wuc2qe Scrolling should hide header text](https://app.clickup.com/t/1wuc2qe)

**Comment by HenryQuillin** [comment]
@j-mateo For some reason, the artist bio will only work as expected (disappears on scroll down) in the Firebase link after I refresh the page. Is this happening for you too?

**Comment by j-mateo** [comment]
> @j-mateo For some reason, the artist bio will only work as expected (disappears on scroll down) in the Firebase link after I refresh the page. Is this happening for you too?

Yeah, it's happening for me too. Does it happen locally for you or only using the firebase link?

File: src/components/Header.tsx [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+      const heightToHideFrom = 37;
+      const winScroll = document.getElementById("content")?.scrollTop;
+  
+      if (winScroll && winScroll > heightToHideFrom) {
+        isTextVisible && setIsTextVisible(false); // to limit setting state only the first time
+      } else {
+        setIsTextVisible(true);
+      }
+    };
+    document && document.getElementById("content")?.addEventListener("scroll", listenToScroll);
```
Comment (j-mateo) [inline src/components/Header.tsx:25]: I would add some logs around these calls to make sure your scroll listener is getting attached

**Comment by HenryQuillin** [comment]
@j-mateo Ok I think I fixed the issue. Can you give a final review before I merge?

**Review by j-mateo — APPROVED** [review]
💪🏽

**Pull request merged**